CTOR-860 : Stream connectors 24.08 Release Note (#3506)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/integrations/stream-connectors.md

## 2024

### August
 

>  <Tabs groupId="sync">

Author: ```suggestion
<Tabs groupId="sync">
```

<TabItem value="Fix" label="Fix">

- [**Splunk Metrics**](data-analytics/sc-splunk-metrics.md) - Fixed an issue with **max_buffer_size** parameter in Splunk stream connector.
- [**Clickhouse**](data-analytics/sc-clickhouse.md) - Added a protection on metric names containing spaces.

> - [**InfluxdDB Metrics**] - Added a protection on metric names containing spaces.

Author: ```suggestion
- **InfluxdDB Metrics** - Added a protection on metric names containing spaces.
```

> - [**Influxdb2 Metrics**](data-analytics/sc-influxdb2-metrics.md) - Added a protection on metric names containing spaces.

Author: ```suggestion
- [**InfluxdDB Metrics**] - Added a protection on metric names containing spaces.
```

## 2024

### August

> 

Author: ```suggestion
 
 <Tabs groupId="sync">
<TabItem value="Fix" label="Fix">

- [**Splunk Metrics**](data-analytics/sc-splunk-metrics.md) - Fixed an issue with **max_buffer_size** parameter in Splunk stream connector.
- [**Clickhouse**](data-analytics/sc-clickhouse.md) - Added a protection on metric names containing spaces.
- [**Influxdb2 Metrics**](data-analytics/sc-influxdb2-metrics.md) - Added a protection on metric names containing spaces.
- [**Centreon library: sc_macro**](https://github.com/centreon/centreon-stream-connector-scripts/blob/develop/modules/centreon-stream-connectors-lib/sc_macro.lua) - Fixed an issue with **_scshort flag** leading to the macro name of an event being displayed instead of its value.

</TabItem>
<TabItem value="Enhancement" label="Enhancement">

- [**Centreon library: sc_params**](https://github.com/centreon/centreon-stream-connector-scripts/blob/develop/modules/centreon-stream-connectors-lib/sc_params.lua) - Added a new **enable_bam_host** parameter to send BAM "services".
- [**Centreon library: sc_flush**](https://github.com/centreon/centreon-stream-connector-scripts/blob/develop/modules/centreon-stream-connectors-lib/sc_flush.lua) - Enhance error report when using **send_data** method in stream connector log files.

</TabItem>
</Tabs>
 
```
